Adding note below release calendar. Can we highlight some rows in calendar? (#9237)
magento/devdocs: src/release/index.md

<sup>\-\- Indicates items that are not applicable to this release.</sup><br>
<sup>1 This is the last patch release for the 2.3.x release line. The 2.3.x release line reaches End of Life (EOL) in April 2022.</sup><br>
<sup>2 There is no full patch release in October 2022.</sup><br>

> Note: Releases highlighted in gray (Patch and Security Patch) are three opportunities for you to upgrade your core code in 2022 to keep your platform secure, reliable and performant. Feature releases will occur every other month. New releases will be available via external module or extension. Updates to independent features already implemented will happen automatically.

Reviewer: We use the following inline attributes for notes on devdocs:

- `{:.bs-callout-info}` - Blue box
- `{:.bs-callout-tip}` - Yellow/orange box
- `{:.bs-callout-warning}` - Green box

In this case, I think `{:.bs-callout-tip}` is a good fit.

> Note: Releases highlighted in gray (Patch and Security Patch) are three opportunities for you to upgrade your core code in 2022 to keep your platform secure, reliable and performant. Feature releases will occur every other month. New releases will be available via external module or extension. Updates to independent features already implemented will happen automatically.

Reviewer: ```suggestion
`{:.bs-callout-tip}`
There are three Patch and Security patch releases in 2022 that are opportunities for you to upgrade to keep your platform secure, reliable, and performant. Feature releases occur every other month. Feature releases are available through external modules or extensions. Updates to independent features that have already been released happen automatically.
```

Author: Like this change! just editing slightly

+ `{:.bs-callout-tip}`
+ Patch and Security Patch Releases are opportunities to upgrade the core codebase to keep your platform secure, reliable, and performant. Feature Releases occur every other month. Feature releases are independent of the core codebase and are available via external module or extension. Any updates to existing independent features will also be released during Feature Release periods and will happen automatically if the feature is already implemented.

Reviewer: Nice! I replaced "via" with "through" because we avoid Latin words/abbreviations in docs.
